.travis.yml

```diff
@@ -50,13 +50,36 @@ script:
         mkdir -p "${HOME}/.foldercache/${exercise}/.stack-work"
         ln -f -s "${HOME}/.foldercache/${exercise}/.stack-work"
 
-        # Here we prepare the exercise to be tested by Stack.
-        MODULE=`sed -n 's/ *module \+\([a-zA-Z0-9]\+\).*/\1/p' src/Example.hs`
-        mv src/Example.hs "src/${MODULE}.hs"
-
-        stack test ${SET_RESOLVER} `# Select the correct resolver. `\
-                   --install-ghc   `# Download GHC if not in cache.`\
-                   --no-terminal   `# Terminal detection is broken.`\
-                   --pedantic      `# Enable -Wall and -Werror.    `
+        if [ -f src/Example.hs ]; then
+            # Here we prepare the exercise to be tested by Stack.
+            MODULE=`sed -n 's/ *module \+\([a-zA-Z0-9]\+\).*/\1/p' src/Example.hs`
+            mv src/Example.hs "src/${MODULE}.hs"
+
+            stack test ${SET_RESOLVER} `# Select the correct resolver. `\
+                       --install-ghc   `# Download GHC if not in cache.`\
+                       --no-terminal   `# Terminal detection is broken.`\
+                       --pedantic      `# Enable -Wall and -Werror.    `
+        else
+            found=
+            for example in examples/*; do
+                echo "testing ${example}"
+                rm -f src/*.hs
+                mv ${example}/*.hs src
+                mv ${example}/package.yaml .
+                found=1
+
+                # ARGH, DUPLICATE CODE
```

Comment:
Don't worry with that for now. Let's just add the features and later we can simplify everything. :+1: